setup.py

```diff
@@ -1,7 +1,13 @@
 #!/usr/bin/env python
 from __future__ import absolute_import
 
-from setuptools import find_packages, setup
+import os.path
+from io import open as open_as_of_py3
+
+from setuptools import setup, find_packages
+
+README = open_as_of_py3(os.path.join(os.path.dirname(__file__), 'README.rst')
```

Comment:
Nit again: closing the parens like this conflicts with our python style guide: https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines (I know, the horror...)

Hence my original suggestion here.

Reply:
okay, that *should* be within guidelines now.  

Will need to take the time to build up new style habits